src/index.js

```diff
@@ -288,6 +328,23 @@ export default class AuthnWidget {
       window.location.replace(state.resumeUrl);
       return;
     }
+    if (currentState === 'EXTERNAL_AUTHENTICATION_REQUIRED') {
+      if (state.presentationMode === 'POP_UP') {
+        if (this.store.getStore().status !== this.store.getPreviousStore().status) {
```

Comment:
I'm not sure I understand this scenario. Is this just so the user can refresh the parent without messing with the current popup?

Reply:
this if for the case when the user closes the popup without authenticating and a `GET_FLOW` is executed again. I do not want to trigger any `window.open` since I want the user to click `Try Again` instead if they want to reopen. The window.open would also trigger a popup that is blocked by the browser as there's no user action associated with it since it was trigger by the popup being closed then `GET_FLOW`

Reply:
Nevermind, this looks like we need it for when the popup is closed but the user has not signed in